jvector-examples/src/main/java/io/github/jbellis/jvector/example/Bench.java

```diff
@@ -131,55 +144,107 @@ private static ResultSummary performQueries(DataSet ds, RandomAccessVectorValues
     }
 
     public static void main(String[] args) throws IOException {
-        System.out.println("Heap space available is " + Runtime.getRuntime().maxMemory());
```

Comment:
please review and either delete commented-out lines, or restore them, before submitting PR